### DISCUSSION TITLE
Why can't I convert an issue into discussion and vice-versa?

### DISCUSSION THREAD

**Discussion by Vadorequest** [community/community#3180] [Discussions] [answered]
I want to convert an issue into a discussion, but I can't. Why is that?

![image](https://user-images.githubusercontent.com/3807458/104935640-c4be0800-59ab-11eb-9f38-2861cda41b35.png)

I enabled GitHub Discussion during beta, maybe that's why?

**Comment by Harmon758** [comment, marked as answer]
See https://github.com/github/feedback/discussions/2924#discussioncomment-254592, https://github.com/github/feedback/discussions/2952, and https://github.com/github/feedback/discussions/3105.
